tools/invoke-tests

```diff
@@ -77,8 +77,8 @@ args=()
 # use unbuffer if it was found so e.g. timestamps on OBS builds will be useful
 # note: When executing tests with prove it seems that the output is only flushed after one test has finished unless there's a terminal.
 [[ $unbuffer_path ]] && [[ $unbuffer_path != 'UNBUFFER_PATH-NOTFOUND' ]] && args+=("$unbuffer_path")
-# split TESTS on white-spaces to allow specifying multiple tests as documented
-TESTS=${TESTS:-"-r t xt"}
+# run tests from t only by default
+TESTS=${TESTS:-"t"}
```

Comment:
ok, I guess that will work. The `-r` would be needed when we have subdirectories with tests though.